module.mk

```diff
@@ -163,3 +163,9 @@ clean: set_up_kernels/clean eager_package/clean
 
 nuke: clean python_env/clean
 	rm -rf $(OUT)
+
+.PHONY: stubs
+stubs:
+	pip install mypy
+	stubgen -m tt_lib -m tt_lib.device -m tt_lib.profiler -m tt_lib.tensor -m tt_lib.dtx -m tt_lib.operations -o tt_eager
```

Comment:
@tt-rkim can we run these commands as part of pre-commit hook?

Reply:
Yep - we should put this into part into static checks. Should we do this for @pavlejosipovic ?

Reply:
I've changed the approach stubs are now generated during build and *.pyi files are in gitignore